.codeclimate.yml

```diff
@@ -0,0 +1,15 @@
+version: "2"
+plugins:
+  radon:
+    enabled: true
+    config:
+      threshold: "C"
+  pep8:
```

Comment:
We might want to import a linter such as flake8 or autopep, and runt that as part of a `test-lint` target that would be a dependency of `test-unit`

Reply:
Sounds like a good idea for a future improvement. These ones here are just official code climate analysis plugins though. https://docs.codeclimate.com/docs/list-of-engines

Reply:
Oh, not for CodeClimate. I meant adding those check to our local toolchain so we can match the PEP8 requirement more easily, without needing a round-trip to CodeClimate.

Reply:
Sure. It's out of scope for this task though.